patroni/postgresql.py

```diff
@@ -521,6 +525,26 @@ def _initialize(self, config):
             self.set_state('initdb failed')
         return ret
 
+    def _custom_bootstrap(self, config):
+        params = ['--scope=' + self.scope, '--datadir=' + self._data_dir]
+        try:
+            logger.info('Running custom bootstrap script: %s', config['command'])
+            if subprocess.call(shlex.split(config['command']) + params) != 0:
+                self.set_state('custom bootstrap failed')
+                return False
+        except Exception:
+            logger.exception('Exception during custom bootstrap')
+            return False
+
+        self._post_restore()
+        self.save_configuration_files()
```

Comment:
_post_restore will can restore_configration_files()? Why do we need to save them right afterwards?